docs/src/pages/mapping.astro

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
This shouldn't be part of the pages dir, put it in the layout dir instead, with a `<slot />` instead of the actual content, and set the layout in the markdown frontmatter. 

You can create a new layout with this and use that layout in the frontmatter for mapping-content.md, then rename mapping-content.md to mapping.md and astro would create a page for it without needing another component just to wrap it.

https://docs.astro.build/en/guides/markdown-content/#frontmatter-layout

Reply:
oof my lack of astro knowledge is showing

i'll get to this after #677 merges